composer.json

```diff
@@ -47,15 +47,16 @@
         "drupal/persistent_login": "~1.0",
         "drupal/phingdrushtask": "dev-7.x-2.x#f4b020f09da3c174fce40f2bd9a9617689f57124",
         "drupal/r4032login": "dev-1.x#391f10d69b9b8c466a2fddac2a6b99f1289e7781",
-        "drupal/rdf_entity": "^1.0",
+        "drupal/rdf_entity": "^1.0.0-alpha17",
```

Comment:
Actually, this, along with the spdx, was appointed by me guided by Claudiu. Why is it bad to have the `^1.0` annotation? And why would it install an older version? We are controlling both of these libraries so every time we make a new release, we should get it by default.

Reply:
The `composer.json` lists the dependency requirements of our project. If we put `^1.0` then we are declaring that we are compatible with all possible 1.x versions of RDF Entity. Since our `minimum-stability` is set to `dev` Composer is allowed to use any version of RDF Entity starting with the very first commit of `8.x-1.x` and all tags including alpha1 and above.

But we are not in fact compatible with anything below `^1.0.0-alpha17` so we should not allow Composer to use anything below this.